CMakeLists.txt

```diff
@@ -56,3 +56,36 @@ add_subdirectory(test)
 if( LLVM_INCLUDE_EXAMPLES )
   add_subdirectory(examples)
 endif()
+
+if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
+  install(DIRECTORY include/mlir include/mlir-c
+    DESTINATION include
+    COMPONENT mlir-headers
+    FILES_MATCHING
+    PATTERN "*.def"
+    PATTERN "*.h"
+    PATTERN "*.td"
```

Comment:
Do you know why the .td are installed?

Reply:
I don't.

Reply:
Let's skip until we have a use-case then?
It'll avoid creating a "Chesterton's fence" :)

(and let's remove .svn as well as .def files, unless we have .def files?)

Reply:
Heh.

Removed .svn. .td, and .def.